diff --git a/node_modules/react-native-slot-machine/index.js b/node_modules/react-native-slot-machine/index.js
index 5bc00d6..b623959 100644
--- a/node_modules/react-native-slot-machine/index.js
+++ b/node_modules/react-native-slot-machine/index.js
@@ -1,27 +1,34 @@
 import React, {Component} from 'react';
-import {View, Text, StyleSheet, Animated, Easing} from 'react-native';
-
+import {View, Text, StyleSheet, Animated, Easing, TouchableOpacity} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
 const styles = StyleSheet.create({
     container: {
         flexDirection: 'row',
         overflow: 'hidden',
     },
     slotWrapper: {
-        backgroundColor: 'gray',
-        marginLeft: 5,
+        // backgroundColor: 'red',
+        marginLeft:10,
+        marginRight:10
+
     },
     slotInner: {
-        backgroundColor: 'black',
+        backgroundColor: '#EBD3B9',
+        // opacity: 0.3,
         alignSelf: 'stretch',
         justifyContent: 'center',
         alignItems: 'center',
         padding: 2,
+        lineHeight:72,
+        borderRadius:10
     },
     text: {
-        fontSize: 50,
+        fontSize: 60,
         top: -2,
-        fontWeight: 'bold',
-        color: '#b5b7ba',
+        fontWeight: '600',
+        color: '#631E00',
+        fontFamily:'Barlow-SemiBold'
+        // opacity:0.7
     },
     innerBorder: {
         position: 'absolute',
@@ -29,8 +36,9 @@ const styles = StyleSheet.create({
         right: 1,
         left: 1,
         bottom: 1,
-        borderColor: 'black',
-        borderWidth: 1,
+        borderColor: '#EFDACF',
+        borderWidth: 5,
+        borderRadius:10,
         zIndex: 1,
     },
     outerBorder: {
@@ -39,8 +47,9 @@ const styles = StyleSheet.create({
         right: 0,
         left: 0,
         bottom: 0,
-        borderColor: '#989898',
+        borderColor: 'rgba(218, 155, 124, 0.3)',
         borderWidth: 1,
+        borderRadius:10,
         zIndex: 1,
     },
     overlay: {
@@ -48,6 +57,7 @@ const styles = StyleSheet.create({
         top: 0,
         right: 0,
         left: 0,
+        // borderRadius:10,
         backgroundColor: '#ffffff77'
     }
 });
@@ -244,7 +254,7 @@ export default class SlotMachine extends Component {
                 {slots}
                 <View style={[styles.innerBorder, overrideStyles.innerBorder]} />
                 <View style={[styles.outerBorder, overrideStyles.outerBorder]} />
-                <View style={[styles.overlay, {bottom: height / 1.6}, overrideStyles.overlay]} />
+                {/* <View style={[styles.overlay, {bottom: height / 1.6}, overrideStyles.overlay]} /> */}
             </View>
         );
     }
@@ -254,6 +264,7 @@ export default class SlotMachine extends Component {
         return (
             <View style={[styles.container, this.props.styles.container]}>
                 {slots}
+                {/* <TouchableOpacity onPress={this.generateSlots}><Text>ty</Text></TouchableOpacity> */}
             </View>
         );
     }
